src/selectors/entities/roles.ts

```diff
@@ -112,7 +115,7 @@ export const getMyTeamPermissions = reselect.createSelector(
     getMyTeamRoles,
     getRoles,
     getMySystemPermissions,
-    (state, options) => options.team,
+    (state: GlobalState, options: any) => options.team,
```

Comment:
We can type easily the permissions options, are always the same, optional `channel` optional `team` and mandatory `permission`